tests/manual-test-cases/Group5-Functional-Tests/5-22-NFS-Volume.robot

```diff
@@ -111,6 +111,14 @@ Gather NFS Logs
     ${out}=  Run Keyword And Continue On Failure  Run  sshpass -p %{DEPLOYED_PASSWORD} ssh -o StrictHostKeyChecking\=no root@${NFS_READONLY_IP} dmesg
     Log  ${out}
 
+Check NFS Connection
+    ${tmpdir1}= Run And Return Output  "mktemp -d"
```

Comment:
while it doesn't really make a difference, you don't need to quote this command and I would prefer not since we don't do that anywhere else...

Reply:
also missing double space between = and Run